CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 6.0.1 - 2021-06-03
+
+#### SASS variable updates
+
+- :warning: `$ui-border-2-light` is now `#d4d4d4` (`$blk-040`)
```

Comment:
maybe note the changes to the resulting css vars if there are any that are updated as well

Reply:
Done! Should I bump the patch version on package-lock/package.json too (6.0.1), or is that done during release?